### DISCUSSION TITLE
Control local Apple Music playback via Airplay

### DISCUSSION THREAD

**Discussion by dunxd** [music-assistant/support#2523] [Feature requests and ideas]
This is not a suggestion relating to the Apple Music subscription service.
I have a large music collection stored on my Mac. From the Apple Music app I can play this to Airplay speakers. From an iPhone, or using the Android app Retune, I can remotely control that playback.

It would be nice of Music Assistant could do this too.

The current way I have found of achieving this is to share the music folder on the Mac, then add this to MA as a filesystem provider, and play this way. However, this doesn't allow me to benefit from playlists or genius feature in the Apple Music app, and I think requires more compute resources on the device running MA.

My overall goal is to have MA as the single interface for playing music in the highest audio quality around my home, playing from any source to any kind of speaker.

**Comment by OzGav** [comment]
> My overall goal is to have MA as the single interface for playing music in the highest audio quality around my home, playing from any source to any kind of speaker.

I’m a bit confused as this is exactly what MA does?

> doesn't allow me to benefit from playlists or genius feature

Yes you would need to recreate your playlists in MA. In future MA is planned to have more dynamic playlists which hopefully will align with what genius does.
